inst/demo-pipe.Rmd

```diff
@@ -424,3 +424,39 @@ sl %>% st_new() %>% st_sizes(row = 2  ) %>% stable() %>% st_asis()
 ```{r}
 data %>% st_new() %>% st_sizes(col = 20) %>% stable() %>% st_asis()
 ```
+
+\clearpage
+
+# Render to long table
+
+```{r}
+longd <- bind_rows(data, data, data, data)
+
+longd %>% 
+  st_new() %>% 
+  st_files(r = "demo-pipe.Rmd") %>%
+  st_notes("This example demonstrates how to make long table") %>%
+  st_args(lt_cap_text = "Adding a caption to the table") %>% 
```

Comment:
How about adding the label here too? That seems to be something people often ask about when making long tables

Reply:
I added this but didn't render the document; would cause a conflict with the PR that Kyle is reviewing.

Reply:
I assume I don't need to wait for that to appear to approve the PR?